src/honeybadger/authentic_handler.cr

```diff
@@ -11,7 +11,7 @@ module Honeybadger
     end
 
     def call(context : HTTP::Server::Context)
-      if user_id = context.session.get? @session_key
+      if (session = context.session) && session.responds_to?(:get?) && (user_id = session.get?(@session_key))
```

Comment:
I ran into this same problem trying to use Honeybadger with Kemal on a small API I have. Except that `session` isn't defined on the HTTP context in Kemal without opting into more

```suggestion
      if context.responds_to?(:session) && session.responds_to?(:get?) && (session = context.session) && (user_id = session.get?(@session_key))
```